config.make.in

```diff
@@ -60,6 +60,11 @@ VERSION = 2.3.98.1
 TARGET = monotouch
 endif
 
+ifeq (x-$(TargetFramework)-$(FSharpCoreBackVersion),x-xamarinwatch-)
+VERSION = 2.3.98.1
```

Comment:
monotouch and monodroid share the same version number because, as far as @7sharp9 and others could determine, the FSharp.Core for these two is identical and reference identically versioned mscorlib etc.

What do we know about Xamarin.Watch?  Looking at the list below it looks like it would be safer and more consistent to give it a platform version number like 2.3.101.1.